tests/unit/core/server/testServer.js

```diff
@@ -31,13 +32,22 @@ describe('core/server/base', () => {
 
   function testClient({ stubRoutes = defaultStubRoutes } = {}) {
     function createStoreAndSagas() {
+      const sagaMiddleware = createSagaMiddleware();
       return {
-        store: createStore(combineReducers({ reduxAsyncConnect })),
-        sagaMiddleware: null,
+        store: createStore(
+          combineReducers({ reduxAsyncConnect }),
+          undefined,
```

Comment:
Whoa, I must have missed this before. Why the `undefined` here (at least over `null` or something else empty?)